src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java

```diff
@@ -394,11 +394,11 @@ private static void assertGVCFHasOnlyOneSample(final String variantPath, final V
             final LinkedHashMap<String, Path> sampleToFilename = new LinkedHashMap<>();
             for ( final String line : lines) {
                 final String[] split = line.split("\\t",-1);
-                if (split.length != 2
-                        || split[0].isEmpty() || containsWhitespace(split[0])
-                        || split[1].isEmpty() || containsWhitespace(split[1])) {
+                if (split.length != 2)
+                    throw new UserException.BadInput("Expected a file of format\nSample\tFile\n but found line: " + line);
+                if ( !split[0].trim().equals(split[0]) || split[0].trim().isEmpty()
+                        || !split[1].trim().equals(split[1]) || split[1].trim().isEmpty())
                     throw new UserException.BadInput("Expected a file of format\nSample\tFile\n but found line: " + line);
```

Comment:
Add curly braces around if statement body.

Reply:
Also update the tool documentation to describe the requirements for the sample name map file (eg., "Sample names in the sample name map file may contain internal non-tab spaces, but may not begin or end with whitespace.")

Reply:
Also, this error message here could be better -- this is the case where either the sample name or the VCF path had leading/trailing whitespace, or was all whitespace. Error message should state that.

Reply:
done, I also removed the whitespace check from the path (because I don't think there should be a failure condition for that come to think of it (because the path API handles it as far as I can tell))